PurchasesCoreSwift/FoundationExtensions/Thread+Extensions.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  Thread+Extensions.swift
+//
+//  Created by Joshua Liebowitz on 8/20/21.
+
+import Foundation
+
+extension Thread {
+
+    var isRunningXCTest: Bool {
```

Comment:
Hey @taquitos, just to know. Is this property used?

Reply:
HA! I added that to test some stuff, then forgot to remove it 🤦‍♂️